lib/rubocop/cop/rails/inquiry.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks that ActiveSupport's `inquiry` method is not used.
```

Comment:
Can you update it?

```suggestion
      # This cop checks that Active Support's `inquiry` method is not used.
```

> The proper names of Rails components have a space in between the words, like “Active Support”. ActiveRecord is a Ruby module, whereas Active Record is an ORM.

https://github.com/rails/rails/blob/master/guides/source/api_documentation_guidelines.md#wording